addons/survey/models/survey_user_input.py

```diff
@@ -272,16 +272,19 @@ def get_print_url(self):
     # CREATE / UPDATE LINES FROM SURVEY FRONTEND INPUT
     # ------------------------------------------------------------
 
-    def save_lines(self, question, answer, comment=None):
-        """ Save answers to questions, depending on question type
+    def _save_lines(self, question, answer, comment=None, overwrite_existing=True):
+        """ Save answers to questions, depending on question type.
 
-            If an answer already exists for question and user_input_id, it will be
-            overwritten (or deleted for 'choice' questions) (in order to maintain data consistency).
+        :param bool overwrite_existing: if an answer already exists for question and user_input_id
+        it will be overwritten (or deleted for 'choice' questions) in order to maintain data consistency.
+        Set to False to just skip if an existing answer exists.
```

Comment:
We could call the new parameter "raise_if_exists", and/or adapt the docstring, as we don't "just skip".